src/main/java/com/google/api/codegen/DiscoveryContext.java

```diff
@@ -213,32 +213,48 @@ protected String getDefaultString(Type type, Field field) {
     return lineEnding(stringLiteral(Strings.nullToEmpty(DefaultString.forPattern(stringPattern))));
   }
 
-  // Line wrap `str`, returning a list of lines. Each line in the returned list is guaranteed to
-  // not have new line characters.
+  // Line wrap `str`, returning a list of lines. Each line in the returned list is guaranteed to not
+  // have new line characters. The first line begins with `firstLinePrefix` (defaults to list
+  // bullet: "* "), while subsequent lines begin with a hanging indent of equal width.
+  public List<String> lineWrapDoc(String str, int maxWidth, String firstLinePrefix) {
```

Comment:
Why do we need to wrap s_lineWrapDoc instead of calling it directly? It seems like it would be less verbose to remove these two wrappers and then s/s_lineWrapDoc/lineWrapDoc/g. Is there a Java subtlety I'm missing?